internal/controller/configmap_controller.go

```diff
@@ -156,21 +158,37 @@ func (r *ConfigMapReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 
 	}
 
-	err := cmlib.AnnotateCM(ctx, r.Client, configMap, internal.InsightsUpdatedAnnotationLabel, time.Now().UTC().Format(time.RFC3339))
-
-	if err != nil {
-		log.Error(err, "Unable to update configmap")
-		return ctrl.Result{}, err
-	}
-
 	// Here we run the post processors
+	retryPostprocessing := false
```

Comment:
So on the retry logic. I'm wondering if it makes sense for us to keep track (on the cm) which parts of the processing has actually succeeded, so that the main push and the various post-processors can skip the retry if _one_ of the post-processors fail?

Reply:
This is inline with my earlier comments about better retry. My understanding is it's fine if the insights get shipped to core more than once, and it's also fine for DT. As the postprocess logic gets more complicated, I think this will be needed, I'm not sure it's worth the effort right now. We can see how this implementation works with our internal projects first.

In theory this is only triggered when the endpoint is down/restarting temporarily.

Reply:
Yeah, I think this is right - it doesn't really matter if we push multiple times - the only thing that might be weird is the updated times reflected in the problems UI, I guess. But, realistically, I wonder how often we'll see this.

Reply:
Didn't make any of these changes for this iteration.